app/boards/shields/eternal_keypad/eternal_keypad_lefty.keymap

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <behaviors.dtsi>
+#include <dt-bindings/zmk/keys.h>
+#include <dt-bindings/zmk/bt.h>
+
+#define BASE        0
+#define ARROW       1
+#define FUNC        2
+
+/ {
+    chosen {
+        zmk,kscan = &kscan0;
+        zmk,matrix_transform = &default_transform;
```

Comment:
Shouldn't this be the lefty transform?

Reply:
In an ideal world, yes.  See the previous comment for details.